src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+
+const promise1 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was resolved!');
+
+  newDiv.className = 'message';
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  logo.addEventListener('click', () => {
+    resolve(body.insertAdjacentElement('afterbegin', newDiv));
+  });
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was rejected!');
+
+  newDiv.className = 'message error-message';
+
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
```

Comment:
The promise `promise2` is set to reject when the logo is clicked after 3 seconds. However, this promise is not being used effectively, as it is not clear how it interacts with `promise1`. Ensure that the promises are used in a way that makes sense for your application's logic.